sdks/builds/llvm.mk

```diff
@@ -123,6 +129,16 @@ TARGETS += llvm-$(1)
 
 endef
 
+# -DLLVM_ENABLE_THREADS=0 is needed because mxe doesn't define std::mutex etc.
+# -DLLVM_BUILD_EXECUTION_ENGINE=Off is needed because it depends on threads
+# -DCROSS_TOOLCHAIN_FLAGS_NATIVE is needed to compile the native tools (tlbgen) using the host compilers
+llvm_win_configure_flags = -DLLVM_ENABLE_THREADS=0 -DCROSS_TOOLCHAIN_FLAGS_NATIVE=-DCMAKE_TOOLCHAIN_FILE=$(LLVM_SRC)/cmake/modules/NATIVE.cmake -DLLVM_BUILD_EXECUTION_ENGINE=Off
```

Comment:
is the attached `-DDCROSS_TOOLCHAIN_FLAGS_NATIVE=-DCMAKE_TOOLCHAIN_FILE` on purpose? And put these values in `_llvm_$(1)_CMAKE_FLAGS` directly at https://github.com/mono/mono/pull/8260/files#diff-e19948cc4c89121d1aa1068fd3d60b01R95